src/main/java/com/crowsofwar/avatar/common/entity/data/Behavior.java

```diff
@@ -96,7 +109,7 @@ public int getId() {
 	 * Called every update tick.
 	 *
 	 * @return Next Behavior. Return <code>this</code> to continue the Behavior.
-	 * Never return null.
+	 *         May Never return null.
```

Comment:
```suggestion
	 *         May never return null.
```
Sorry, I made a typo.